### ISSUE TITLE
Keyboard navigation in saved colors: unable to reach action buttons

### ISSUE THREAD

**Issue by o-alquimista** [josephmawa/Bella#22]
This is a recording of me pressing TAB and arrow keys to navigate through the table.

https://github.com/user-attachments/assets/743e5fd4-f712-4e2c-a17a-76a50de70a05

1. You'll see that I can't even reach the delete buttons — they're completely unreachable.

2. If you Shift+TAB your way backwards, you can reach the "View details" button — it's the only way to reach it.

**Comment by josephmawa** [comment]
Hi @o-alquimista 

Thanks for catching this. It is also happening on my end. I will have to investigate why it is like this.

**Comment by o-alquimista** [comment]
Cool. Let me know if you need help testing it.
